README.md

```diff
@@ -24,21 +24,39 @@ The higher order goal to keep in mind for any tools developed here should be to
 time to resolution for issues that are signaled by anything present in the CI data. There are a lot of ways that this
 could be done, and instead of pre-defining a specific problem to solve, our immediate thinking is to make the initial
 tools and relevant data as accessible as possible to foster collaboration and contributions between data scientists and
-OpenShift engineers.
+DevOps engineers.
 
-## First Step/ Ways to get Involved:
+## Current Work
 
-**1 )** Exploratory data analysis of available data sources (gcsweb, TestGrid, Sippy) to understand the different data
-types that need to be addressed and identify what kind of ML approaches are possible.
+* Interactive and reproducible notebooks for the [entire project](https://github.com/aicoe-aiops/ocp-ci-analysis) are available at the [operate-first jupyterhub](https://jupyterhub-opf-jupyterhub.apps.cnv.massopen.cloud/hub/login)
```

Comment:
should this be rephrased to `are available as an image on [jupyterhub](https://jupyterhub-opf-jupyterhub.apps.cnv.massopen.cloud/hub/login) deployed on the MOC cluster`

Reply:
@oindrillac :+1: